be/src/olap/tablet_meta.cpp

```diff
@@ -172,8 +172,8 @@ const RowsetMeta* TabletMeta::get_inc_rowset(const Version& version) const;
     return rs_meta;
 }
 
-NewStatus TabletMeta::modify_rowsets(const vector<RowsetMeta>& to_add,
-                                     const vector<RowsetMeta>& to_delete) {
+NewStatus TabletMeta::modify_rowsets(const vector<std::shared_ptr<RowsetMeta>>& to_add,
+                                     const vector<std::shared_ptr<RowsetMeta>>& to_delete) {
```

Comment:
I think rowsets_to_add and rowsets_to_delete  is better name

Reply:
Name is long may be not better, because line may be long.